modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/_data_set_display.scss

```diff
@@ -41,25 +41,11 @@
 		}
 	}
 
-	.main-input-wrapper {
-		position: relative;
-	}
-
 	.main-input-search {
```

Comment:
I think this rule-set can be removed. I don't see this class being used after update.

Reply:
Yes! Thanks for point up this. Removing...